deploy/olm-catalog/ibm-monitoring-grafana-operator/1.14.0/ibm-monitoring-grafana-operator.v1.14.0.clusterserviceversion.yaml

```diff
@@ -333,6 +333,18 @@ spec:
                 - update
                 - patch
                 - delete
+            - apiGroups:
+                - networking.k8s.io
```

Comment:
can we club the networking.k8s.io with existing apiGroup instead of creating a set of new role again?

- apiGroups:
               - extension
                - networking.k8s.io

Reply:
I have just added it as a another array element entry. because i wanted to remove alteaset `create` verb on `extensions` apiGroup going forward and doesn't alter any verb on `networking.k8s.io`